typings/index.d.ts

```diff
@@ -187,7 +187,7 @@ export interface ContextMessageUpdate extends Context {
    * @param extra Additional params to send message
    * @returns a Message on success
    */
-  replyWithHTML(html: string, extra?: tt.ExtraEditMessage): Promise<tt.Message>
+  replyWithHTML(html: string, extra?: tt.ExtraReplyMessage): Promise<tt.Message>
```

Comment:
`replyWithHTML()` is a wrapper for `reply()`, it only adds `parse_mode: 'HTML'` to `extra` parameter, therefore `extra` should be of the same type as `reply()` has.